src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/Multiplatform/StatusBar/StatusBar.android.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Android.OS;
+using Android.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform
+{
+	public static partial class StatusBar
+	{
+		static partial void PlatformSetColor(Color color)
+		{
+			if (Build.VERSION.SdkInt < BuildVersionCodes.M)
+				return;
```

Comment:
How about adding a log message that this feature isn't supported in older versions? I believe that will improve the dev xp.

Reply:
Like `Debug.WriteLine`, or `Trace.TraceInformation`, or something different?

Reply:
There's a Log class in `Xamarin.Forms.Internals` that we can use.

Reply:
Searched through the project. We use `Debug.WriteLine` (`Trace` or `Log`) are never used. So added logs as `Debug.WriteLine`